paysage/optimizers.py

```diff
@@ -2,17 +2,55 @@
 from . import backends as B
 
 
+# ----- LEARNING RATE SCHEDULERS ----- #
+
+class Scheduler(object):
+
+    def __init__(self):
+        self.lr = 1
+        self.iter = 0
+        self.epoch = 0
+
+    def increment(self, epoch):
+        self.iter += 1
+        self.epoch = epoch
+
+
+class ExponentialDecay(Scheduler):
+
+    def __init__(self, lr_decay=0.9):
+        super().__init__()
+        self.lr_decay = lr_decay
+
+    def get_lr(self):
+        self.lr = (self.lr_decay ** self.epoch)
+        return self.lr
+
+
+class PowerLawDecay(Scheduler):
+
+    def __init__(self, lr_decay=0.1):
+        super().__init__()
+        self.lr_decay = lr_decay
+
+    def get_lr(self):
+        self.lr = 1 / (1 + self.lr_decay * self.epoch)
```

Comment:
did you want `self.lr_decay` in the numerator here?  Otherwise you're always starting from a learning rate of 1.